src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java

```diff
@@ -1153,6 +1160,20 @@ public OpenLoadFlowParameters setVoltageTargetPriorities(List<String> voltageTar
         return this;
     }
 
+    /**
+     * Only if transformer voltage control is active and with mode `AFTER_GENERATOR_VOLTAGE_CONTROL`. Set the nominal
+     * voltage under which the generator voltage control are disabled during outer loop. This parameter overrides the
+     * automatic nominal voltage computation if > 0.
+     */
```

Comment:
@annetill  the ">0" contradicts the parameter doc where auto is for <0.  Setting the parameter to 0, which means keeping all generators in tension control is interesting and should be allowed.